xep-0403.xml

```diff
@@ -61,32 +76,49 @@
     <li>The mechanism must work cleanly for participants with multiple clients.</li>
     <li>Standard presence messages must be used to share presence.</li>
     <li>Nick changes should be visible as changes (and not as a new user).</li>
-    <li>Where &xep0404; is not used,  participants must be able to directly contact other participants.</li>
+    <li>Where &xep0404; (MIX-ANON) is not used,  participants must be able to directly contact other participants.</li>
+    <li>IQ stanzas, including vCard, can be communicated through a channel.</li>
 
   </ol>
 
 </section1>
 
 <section1 topic='Concepts' anchor='concepts'>
+  
+ <section2 topic="Participant Information in Presence" anchor="concept-participant-info">
+   <p>
+     A MIX channel MAY distribute presence information about channel participants.   In order to share JID and Nick information about a participant, this information is encoded in the presence message.   This allows full presence information to be shared for each participant without the need for the client to perform any lookup.    
+   </p>
+ </section2>
+  
+  
 
-
+<section2 topic="Partipant JID Addressing for Presence and IQ relay" anchor="concept-jid-address">
+  <p>
+    Participants in Presence messages and in IQ messages relayed through a channel are identified by an encoded JID of the form are encoded, using the format "&lt;stable-participant-identifier&gt;#&lt;channel&gt;@&lt;mix domain&gt;".   This provides a stable JID for each participant. This JID format is used in three places:
+  </p>
+  <ol>
+    <li>The 'from' of presence stanzas generated by the channel.</li>
+    <li>The 'to' of IQ stanzas being sent to the channel.</li>
```

Comment:
This is wrong isn't it? If you're sending to the channel you're using the channel's JID.